src/compiler/generate-common-tests.xsl

```diff
@@ -639,7 +639,25 @@
 
          <!-- Replace x:like with specified scenario's child elements -->
          <xsl:when test="self::x:like">
-            <xsl:apply-templates select="key('scenarios', x:label(.))/element()" mode="#current" />
+            <xsl:variable name="label" as="element(x:label)" select="x:label(.)" />
+            <xsl:variable name="scenario" as="element(x:scenario)*" select="key('scenarios', $label)" />
+            <xsl:choose>
+               <xsl:when test="empty($scenario)">
+                  <xsl:sequence select="error(xs:QName('x:XSPEC009'),
+                     concat(name(), ': Scenario not found: ', $label))" />
+               </xsl:when>
+               <xsl:when test="$scenario[2]">
+                  <xsl:sequence select="error(xs:QName('x:XSPEC010'),
+                     concat(name(), ': Multiple scenarios found: ', $label))" />
```

Comment:
"Multiple scenarios found with same label" prevents the message from sounding as if it might be about the referenced scenario containing multiple scenarios as children.

Also, would it be useful to include `count($scenario)` in the message?

Reply:
ec998ed0758fac9ea2581d7385fdc73b1cfb3ff0 updated the message to "# scenarios found with same label".